NOTICE.txt

```diff
@@ -4659,11 +4659,11 @@ Contents of probable licence file $GOMODCACHE/github.com/oklog/ulid@v1.3.1/LICEN
 
 --------------------------------------------------------------------------------
 Dependency : github.com/open-telemetry/opentelemetry-collector-contrib/exporter/fileexporter
```

Comment:
@andrzej-stencel  this we need to update as well when we change versions in go.mod otherwise `check-ci` steps fails.
you can update it by running `mage notice`